BravoLights/Config.ini

```diff
@@ -366,3 +366,39 @@ LowOilPressure = A:ELECTRICAL MASTER BATTERY, bool == 1 AND A:GENERAL ENG OIL PR
 ParkingBrake = A:ELECTRICAL MASTER BATTERY, bool == 1 AND A:BRAKE PARKING POSITION, percent > 1
 ; From model/GB_int.xml
 LowVolts = A:ELECTRICAL MASTER BATTERY, bool == 1 AND A:ELECTRICAL MAIN BUS VOLTAGE, volts < 20
+
+; support for Aerosoft CRJ serie
+[Aircraft.Aerosoft_CRJ_550, Aircraft.Aerosoft_CRJ_700, Aircraft.Aerosoft_CRJ_900, Aircraft.Aerosoft_CRJ_1000]
+MasterWarning = L:ASCRJ_GSF_MASTER_WARN_ON == 1
+MasterCaution = L:ASCRJ_GSF_MASTER_CAUT_ON == 1
+
+; map the button lights to the FCP leds. Modes that are active and armed will lighted and match the FCP leds.
+HDG = L:ASCRJ_FCP_HDG_LED == 1
+NAV = L:ASCRJ_FCP_NAV_LED == 1
+APR = L:ASCRJ_FCP_APPR_LED == 1
+REV = L:ASCRJ_FCP_BC_LED == 1
+ALT = L:ASCRJ_FCP_ALT_LED == 1
+VS = L:ASCRJ_FCP_VS_LED == 1
+IAS = L:ASCRJ_FCP_SPEED_LED == 1
+
+; lights if any of the hydrolics pumps is off. coulnt find any lvar with the pressure itself.
+; 
+LowHydPressure = L:ASCRJ_HYDR_PUMP_1 == 1 OR L:ASCRJ_HYDR_PUMP_2 == 1 OR L:ASCRJ_HYDR_PUMP_3A == 0 OR L:ASCRJ_HYDR_PUMP_3B == 1
+
+; turn on the door lights if any of the doors is open
+Door = L:DOOR_MAIN_DOOR_POS != 0 OR L:DOOR_FWD_CARGO_POS  != 0 OR L:DOOR_AFT_CARGO_POS != 0 OR L:DOOR_SERVICE_POS != 0
+
+; APU on
+APU = L:ASCRJ_APU_STARTSTOP == 1
+
+; mapped to the captain glareshield Engine fire light
+EngineFire = L:ASCRJ_GSC_ENG_FIRE_ON == 1
+
+StarterEngaged = L:ASCRJ_ENG_R_STARTING == 1 OR L:ASCRJ_ENG_L_STARTING == 1
+ParkingBrake = L:ASCRJ_PARK_BRAKE == 1
```

Comment:
Another configuration I received has
`L:ASCRJ_ELEC_BATTMASTER == 1 AND L:ASCRJ_PARK_BRAKE == 1`
here. I think that's simply ensuring that the parking brake warning is only lit if the battery is on, which makes sense to me.